src/node/operator_event.rs

```diff
@@ -1,9 +1,17 @@
 use std::cmp::Ordering;
+use std::fmt;
 
 use crate::dataflow::Timestamp;
 
-/// Operator executors create events upon receipt of messages and watermarks. These events
-/// are processed in the order of their priority and timestamp.
+/// `OperatorEvent` is a structure that encapsulates a particular invocation of the
+/// callback in response to a message or watermark. These events are processed according to the
+/// partial order defined by the `PartialOrd` trait. 
+///
+/// The event is passed to an instance of 
+/// [`OperatorExecutor`](../operator_executor/struct.OperatorExecutor.html)
+/// which is in charge of inserting the event into a
+/// [`ExecutionLattice`](../lattice/struct.ExecutionLattice.html). The `ExecutionLattice` ensures
```

Comment:
```suggestion
/// [`ExecutionLattice`]. The [`ExecutionLattice`] ensures
```

Reply:
Apparently this doesn't work. I guess you need to have imported the module for it to auto-resolve in the docs.

Reply:
Maybe this works?
```suggestion
/// [`ExecutionLattice`](crate::node::lattice::ExecutionLattice). The [`ExecutionLattice`](crate::node::lattice::ExecutionLattice) ensures
```